F/figtree/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "figtree"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://sourceforge.net/projects/figtree/files/figtree-0.9.3.zip", "f0b39cef8a0ab56075cce0c656b1b4327d9bd1acb31aad927241974ac4d10d82")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd figtree-0.9.3/
+make
+cd lib
+mv libann_figtree_version.so $prefix/libann_figtree_version.so
+mv libfigtree.so $prefix/libfigtree.so
+exit
```

Comment:
Libraries need to install to `${libdir}`: https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables :wink: 
```suggestion
cd $WORKSPACE/srcdir/figtree-*/
make
mv "lib/libann_figtree_version.${dlext}" "${libdir}/libann_figtree_version.${dlext}"
mv "lib/libfigtree.${dlext}" "${libdir}/libfigtree.${dlext}"
```